plugins/ssl/ssl-certificate-expiry

```diff
@@ -123,10 +126,33 @@ print_expire_days() {
     [ -n "$starttls" ] && s_client_args="-starttls $starttls"
 
     # shellcheck disable=SC2086
+
+    # The original code only extracted the server certificate,
```

Comment:
Feel free to omit the reference to the previous code. The future reader of this plugin's code is probably not aware of its history ...

Reply:
You are right. That make sense.